README.md

```diff
@@ -111,3 +111,22 @@ yarn ts-node module-ops.ts \
   disable-module \
   --previous-module <prev-module>
 ```
+
+### Module operations with cast
+
+#### Enable module
+
+```
+cast calldata "enableModule(address)" <module-address>
+```
+
+use this calldata to send a transaction from a safe to itself to enable the module on that safe.
+
+#### Disable module
+
+```
+cast calldata "disableModule(address,address)" <previous-module> <module-to-disable>
+```
+
+use this calldata to send a transaction from a safe to itself to disable the module on that safe.
+Or, use the safe UI settings page to disable the module if you do not know the address of `<previous-module>`
```

Comment:
I think we could more easily just say use the UI for this, and omit the `cast` instructions. No need to reinvent the wheel :upside_down_face: